crates/rolldown/src/dev/types/task_input.rs

```diff
@@ -4,6 +4,8 @@ use rolldown_utils::indexmap::FxIndexSet;
 
 #[derive(Debug)]
 pub enum TaskInput {
+  /// Retry initial build after failure
+  InitialBuild,
   /// Initial full rebuild from scratch
   FullRebuild,
```

Comment:
It seems `FullRebuild` is only created at this line: https://github.com/rolldown/rolldown/blob/eee2b98e0f6ff8abd09d22bf258434686ab2e4c3/crates/rolldown/src/dev/bundle_coordinator.rs#L65
Do we need to differentiate `InitialBuild` and `FullRebuild`? It feels like it's the same concept.